app/assets/javascripts/reservations.js

```diff
@@ -9,10 +9,10 @@ $(document).ready(function() {
     if (typeof minDaysFromNow == "undefined") {
       window['minDaysFromNow'] = 0;
     }
-    $("#datepicker").datepicker({'minDate': minDaysFromNow, 'maxDate': maxDaysFromNow});
+    $("#datepicker").datepicker();
 
     $('.datepicker').each(function(){
-      $(this).datepicker({'minDate': minDaysFromNow, 'maxDate': maxDaysFromNow})
```

Comment:
This is also used by the user reservation form, so we can't eliminate these options w/o changing to the `__data` version (probably not worth it right now).

I think you can probably do something for `maxDaysFromNow` like what we did for `minDaysFromNow` so that it's at least defined on the pages that don't use it.